src/SignalR/clients/java/signalr/src/test/java/com/microsoft/signalr/HubConnectionTest.java

```diff
@@ -1494,7 +1494,7 @@ public void afterSuccessfulNegotiateConnectsWithLongPollingTransport() {
         assertEquals("{\"protocol\":\"json\",\"version\":1}" + RECORD_SEPARATOR, sentMessages[0]);
     }
 
-    @Test
+    // Github issue:https://github.com/aspnet/AspNetCore/issues/8262
```

Comment:
Can you use `@Ignore` ? http://junit.sourceforge.net/javadoc/org/junit/Ignore.html

Reply:
Like we use `Skip` in xUnit.net:

```java
@Ignore("https://github.com/aspnet/AspNetCore/issues/8262")
@Test
...
```